test/test_bson_corpus.py

```diff
@@ -17,10 +17,17 @@
 import binascii
 import functools
 import glob
-import json
 import os
 import sys
 
+if sys.version_info[:2] == (2, 6):
+    try:
+        import simplejson as json
+    except ImportError:
+        import json
+else:
+    import json
+
```

Comment:
Can you add `sys.path[0:0] = [""]` to allow us to run the test directly? Without it:
```
$ python test/test_bson_corpus.py -v
Traceback (most recent call last):
  File "test/test_bson_corpus.py", line 32, in <module>
    from test import unittest
ImportError: cannot import name 'unittest'
```

With it:
```
$ python test/test_bson_corpus.py -v
test_all_bson_types (__main__.TestBSONCorpus) ... ok
test_array (__main__.TestBSONCorpus) ... ok
test_binary (__main__.TestBSONCorpus) ... ok
test_boolean (__main__.TestBSONCorpus) ... ok
test_code (__main__.TestBSONCorpus) ... ok
test_code_w_scope (__main__.TestBSONCorpus) ... ok
test_datetime (__main__.TestBSONCorpus) ... ok
test_dbpointer (__main__.TestBSONCorpus) ... ok
test_dbref (__main__.TestBSONCorpus) ... ok
test_document (__main__.TestBSONCorpus) ... ok
test_double (__main__.TestBSONCorpus) ... ok
test_int32 (__main__.TestBSONCorpus) ... ok
test_int64 (__main__.TestBSONCorpus) ... ok
test_maxkey (__main__.TestBSONCorpus) ... ok
test_minkey (__main__.TestBSONCorpus) ... ok
test_null (__main__.TestBSONCorpus) ... ok
test_oid (__main__.TestBSONCorpus) ... ok
test_regex (__main__.TestBSONCorpus) ... ok
test_string (__main__.TestBSONCorpus) ... ok
test_symbol (__main__.TestBSONCorpus) ... ok
test_timestamp (__main__.TestBSONCorpus) ... ok
test_top (__main__.TestBSONCorpus) ... ok
test_undefined (__main__.TestBSONCorpus) ... ok

----------------------------------------------------------------------
Ran 23 tests in 0.037s

OK
```